menyer/hellskitchen/index.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html lang="sv">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Hells Kitchen</title> 
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <header>
+        <h1>Hells Kitchen Meny</h1>
+    </header>
+    <main>
+        <div>
+            <h2>Förrätter</h2>
+            <article>
+                <img src="rakmacka.jpg" alt="logo">
+                <p>Räkmacka- med hemmagjort surdegs bröd <br> 95 kr</p>
+                </article>
+            <article>
+                <img src="vitlöksbröd.jpg" alt="logo">
```

Comment:
Usually, non-english characters and spaces are avoided for file names.

Reply:
okay :)